packages/nuxeo-web-ui-ftest/features/step_definitions/browser.js

```diff
@@ -71,6 +71,10 @@ Then('I can see the selection toolbar', function() {
   this.ui.browser.selectionToolbar.waitForVisible();
 });
 
+Then('I cannot see the display selection link', function() {
+  this.ui.browser.waitForNotVisible('.selectionLink').should.be.true;
+});
```

Comment:
Since the selection link is inside the selection toolbar, it would be more consistent (and efficient) to narrow down the search to the selection toolbar. Something like this:
```suggestion
Then('I cannot see the display selection link', function() {
  this.ui.browser.selectionToolbar.waitForNotVisible('.selectionLink').should.be.true;
});
```